frontend/src/app/modules/home/add-dashboard/add-dashboard.component.sass

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap')
+
+input[type="text"], select
+    width: 100%
+    padding: 12px 20px
+    margin: 8px 0
+    border: 1px solid #ccc
+    border-radius: 4px
+
+.button
+    margin: 5px auto
+    display: block
+    width: 100%
+    font-family: "DM Sans", serif
+    font-weight: 520
+    padding: 12px 16px
+    font-size: 16px
+    border: none
+    cursor: pointer
+    transition: 200ms all ease-in-out
+    color: #000
+
+.disabled
+    color: rgb(0, 0, 0, 0.5)
+
+.cancel-button
+    background-color: #F2F3F4
+    &:hover
+        opacity: 0.7
+
+.create-button
+    background-color: #2ECC71
+    &:hover
+        opacity: 0.8
```

Comment:
Please do not override styles for common elements. If you really need it then override it globally (src/styles folder), but I don't think it is the case.